bonfire/bonfire.py

```diff
@@ -702,6 +702,9 @@ def _cmd_namespace_release(namespace, force, local):
     if not has_ns_operator():
         _error(NO_RESERVATION_SYS)
 
+    if not namespace:
+        namespace = get_current_namespace()
```

Comment:
I think we need to handle the case where `get_current_namespace` returns `None` here and print a fatal error message via the `_error()` func?

Reply:
Good call! Added.